netaddr_test.go

```diff
@@ -357,3 +423,15 @@ func BenchmarkIPv6(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkIPv4Contains(b *testing.B) {
+	b.ReportAllocs()
```

Comment:
also add a test that checks allocs (testing.AllocsPerRun) to make sure it's 0 else fails?